src/helpers/object.ts

```diff
@@ -0,0 +1,22 @@
+/** *****************************************************************************
+ * Copyright 2009-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+import { diff } from "deep-object-diff";
+
+export function hasChanges<T extends object>(obj1: T, obj2: T) {
+	console.log(Object.keys(diff(obj1, obj2)));
+	
+	return Object.keys(diff(obj1, obj2)).length > 0;
```

Comment:
we use lodash in this project and it has isEqual functions whice performs deep comparison